index.js

```diff
@@ -29,6 +22,13 @@ function melodicalBeep(val, cb) {
 }
 
 module.exports = function (val, cb) {
+    
+    if (!process.stdout.isTTY ||
+        process.argv.indexOf('--no-beep') !== -1 ||
+        process.argv.indexOf('--beep=false') !== -1) {
+        return;
+    }
+    
```

Comment:
Wrong indentation and you also left some trailing whitespace in there, but I'll fix when merging.